.github/workflows/check-crates.yaml

```diff
@@ -0,0 +1,81 @@
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - contracts/**/*
+
+name: check crates
+
+env:
+  RUST_FMT: nightly-2023-04-01
+  RUST_VERSION: "1.67"
+  CARGO_CONCORDIUM_VERSION: "3.0.0"
+
+jobs:
+  rustfmt:
+    name: format
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        crates:
+          - contracts/concordium-governance-committee-election/Cargo.toml
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

Comment:
Please use @v4 for new stuff. This one is deprecated and produces a bunch of warnings.